.gitignore

```diff
@@ -97,3 +97,7 @@ libs/
 .classpath
 .project
 .settings
+
+# Carthage
+**/Carthage/Checkouts/
```

Comment:
Just to be 100% sure: `Carthage/Build` should be included, right?

Reply:
Yes. Carthage/Build is mean to meant to be committed as the carthage-gradle-plugin reads information directly from the built artifacts